frontend/hub/repositories/hooks/useRepositoriesColumns.tsx

```diff
@@ -10,10 +12,53 @@ export function useRepositoriesColumns(_options?: {
   const { t } = useTranslation();
   const tableColumns = useMemo<ITableColumn<Repository>[]>(
     () => [
-      { header: t('Name'), cell: (repository) => <TextCell text={repository.name} /> },
       {
-        header: t('Description'),
-        cell: (repository) => <TextCell text={repository.description} />,
+        header: t('Repository name'),
+        cell: (repository) => <TextCell text={repository.name} />,
+        sort: 'name',
+      },
+      {
+        header: t('Labels'),
+        cell: (repository) => {
+          if (Object.keys(repository.pulp_labels).length == 0) {
```

Comment:
There was a PR merged updating linter rules related to usage of loose equality, if this one is rebase will get the latest eslint rules.

 https://github.com/ansible/ansible-ui/pull/1265